tfhe-fft/Cargo.toml

```diff
@@ -0,0 +1,57 @@
+[package]
+name = "tfhe-fft"
+version = "0.6.0"
+edition = "2021"
+authors = ["sarah el kazdadi <sarah.elkazdadi@zama.ai>"]
```

Comment:
Should we keep Sarah's name ? Or at least change the email address.

Reply:
We could change it same as in the TFHE-rs toml, I'll do that then

Reply:
no authors lol